pandas/plotting/_core.py

```diff
@@ -2567,6 +2567,12 @@ def boxplot_frame_groupby(grouped, subplots=True, column=None, fontsize=None,
     figsize : A tuple (width, height) in inches
     layout : tuple (optional)
         (rows, columns) for the layout of the plot
+    sharex :
```

Comment:
Could you take a look at http://pandas.pydata.org/pandas-docs/stable/contributing_docstring.html for how to format these? Something like

```
sharex : bool, default False
    Whether the horizontal axes is shared among subplots
```

Reply:
Ok, did that. Should this part of the doc-string then be corrected as well?

    subplots :
        * ``False`` - no subplots will be used
        * ``True`` - create a subplot for each group